Source/ACE/Command/Handlers/AdminCommands.cs

```diff
@@ -1001,8 +1001,26 @@ public static void HandleCI(Session session, params string[] parameters)
                 ChatPacket.SendServerMessage(session, "Not a valid weenie id - must be a number between 0 -65,535 ", ChatMessageType.Broadcast);
                 return;
             }
-            var loot = LootGenerationFactory.CreateTestWorldObject(session.Player, weenieId);
-            session.Player.HandleAddToInventory(loot);
+
+            // TODO: we have to be able to identify containers - you can create a wo and see if it has
+            // an item capacity but that seems a bit wasteful.   TBD on best method.   Limited number so
+
+            if (weenieId == 136)
+            {
+                var loot = LootGenerationFactory.CreateTestContainerObject(session.Player, weenieId);
+                session.Player.AddToInventory(loot);
```

Comment:
The main game loop has been staged.  There is still paralleism within the system.  For instance, network packed are handled asyncrhonously from the main game loop.  The commands all race with the worldobjects, so we handle that asynchrony by passing messages (in the form of actions).

If this winds up being a big issue, I may move command handlers into the main thread, but for now I would like to try it this way.